### ISSUE TITLE
[UX2.0] sdwan_transport_wan_vpn_profile_parcel fails with Invalid Format primaryDnsAddressIpv6/secondaryDnsAddressIpv6

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#210]
The following resource:
```
  + resource "sdwan_transport_wan_vpn_profile_parcel" "transport_wan_vpn_profile_parcel" {
      + description                         = "VPN 0 configuration"
      + enhance_ecmp_keying                 = true
      + feature_profile_id                  = "30e53bfc-227c-437b-ae1d-825201f2b93b"
      + id                                  = (known after apply)
      + ipv4_static_routes                  = [
          + {
              + gateway         = "nextHop"
              + network_address = "0.0.0.0"
              + next_hops       = [
                  + {
                      + address_variable = "{{vpn0_ipv4_default_route_nexthop1_ip}}"
                    },
                  + {
                      + address_variable = "{{vpn0_ipv4_default_route_nexthop2_ip}}"
                    },
                ]
              + subnet_mask     = "0.0.0.0"
            },
        ]
      + name                                = "wan_vpn"
      + new_host_mappings                   = [
          + {
              + host_name            = "vbond.local"
              + list_of_ip_addresses = [
                  + "10.0.0.1",
                  + "10.0.0.2",
                ]
            },
        ]
      + primary_dns_address_ipv4_variable   = "{{vpn0_dns_primary}}"
      + secondary_dns_address_ipv4_variable = "{{vpn0_dns_secondary}}"
      + services                            = [
          + {
              + service_type = "TE"
            },
        ]
      + version                             = (known after apply)
      + vpn                                 = 0
    }
```

fails with:
```
│ Error: Client Error
│ 
│   with module.sdwan.sdwan_transport_wan_vpn_profile_parcel.transport_wan_vpn_profile_parcel["transport-wan_vpn"],
│   on ../terraform-sdwan-nac-sdwan/sdwan_profile_parcels.tf line 431, in resource "sdwan_transport_wan_vpn_profile_parcel" "transport_wan_vpn_profile_parcel":
│  431: resource "sdwan_transport_wan_vpn_profile_parcel" "transport_wan_vpn_profile_parcel" {
│ 
│ Failed to configure object (POST), got error: HTTP Request failed: StatusCode 400,
│ {"error":{"message":"Invalid Json Payload Input","code":"SCHVALID0001","details":"{\"Validation
│ Errors\":{\"Invalid Format
│ Attributes\":[\"data.dnsIpv6.primaryDnsAddressIpv6\",\"data.dnsIpv6.secondaryDnsAddressIpv6\"]}}","type":"error"}}
```

It looks like now if IPv6 DNS is not configured, TF pushes following JSON to SD-WAN Manager:
```
         "dnsIpv6":{
            "primaryDnsAddressIpv6":{
               "optionType":"global",
               "value":""
            },
            "secondaryDnsAddressIpv6":{
               "optionType":"global",
               "value":""
            }
         },
```

The correct behavior would be if there is no IPv6 DNS configured, TF should omit dnsIpv6 in JSON payload. We should also make sure same behavior applies if IPv4 DNS is not configured (dnsIpv4).

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/af3abc921ba94cbb6a325f53415e869b2412d32f

**Comment by tzarski0** [comment]
Fixed in v0.3.8 release.
